sdk/nodejs/Makefile

```diff
@@ -67,7 +67,7 @@ unit_tests:: $(TEST_ALL_DEPS)
 	yarn run nyc -s mocha 'bin/tests_with_mocks/**/*.spec.js'
 
 test_auto:: $(TEST_ALL_DEPS)
-	yarn run nyc -s mocha --timeout 300000 'bin/tests/automation/**/*.spec.js'
+	yarn run nyc -s mocha --timeout 900000 'bin/tests/automation/**/*.spec.js'
```

Comment:
I had to bump the timeout here. For each Automation API language (Node.js, Python, and Go), there are two "lifecycle" tests, each take ~5 minutes to run waiting on the remote deployments, for a total of ~10 minutes per language.

1. `RemoteWorkspace.createStack`
2. `RemoteWorkspace.createOrSelectStack`

https://github.com/pulumi/pulumi/blob/d3e6db54eaea8ac0c5ab6bf50a7058d374d05533/sdk/nodejs/tests/automation/remoteWorkspace.spec.ts#L239-L278

Both of those tests kick off `up`, `preview`, `refresh`, and `destroy` deployments.

https://github.com/pulumi/pulumi/blob/d3e6db54eaea8ac0c5ab6bf50a7058d374d05533/sdk/nodejs/tests/automation/remoteWorkspace.spec.ts#L381-L426

Note that these tests only run when `PULUMI_ACCESS_TOKEN` and `PULUMI_TEST_DEPLOYMENTS_API` are set.

If the added time is impactful for CI, we could consider some mitigations:
1. Only run during merge (not sure this will work with coverage)
2. Cut down the tests. We could get away with only one of `createStack` or `createOrSelectStack`, and within those we could cut out the `preview` and `refresh` operations, and still have decent test coverage. That should cut the total time from ~10 min per language to ~2.5 min per language.